docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
+        </Layer>}
+    </ScrollBoundaryContainer>
   )
-};
+}
 `}
-  />,
-);
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+function SemiTransparentWhiteButtonExample() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+  const viewRef = React.useRef();
 
-card(
-  <Example
-    name="Width"
-    id="widths"
-    defaultCode={`
-<React.Fragment>
-  <Box padding={2}>
-    <Button text="Default inline button" />
-  </Box>
-  <Box padding={2}>
-    <Button fullWidth text="Full-width button" />
-  </Box>
-</React.Fragment>`}
-  />,
-);
+  React.useEffect(() => {
+    setOpen(true)
+  });
 
-card(
-  <Example
-    name="Icons"
-    id="iconEnd"
-    defaultCode={`
-    <Button iconEnd="download" text="Download CVS file" />
+  return (
+    <ScrollBoundaryContainer>
+      <Box ref={viewRef} width={300} height={220}>
+        <Box display="flex" justifyContent="center" ref={anchorRef}>
+        <Tabs
+          activeTabIndex={1}
+          onChange={() => {}}
+          tabs={[{ href: '#Anchor', text: 'Created'}]}
+        />
+        </Box>
+      </Box>
+      {open &&
+        <Layer>
+          <Popover
+            anchor={anchorRef.current}
+            color="blue"
+            idealDirection="down"
+            showCaret
+            onDismiss={() => {}}
+            positionRelativeToAnchor={false}
+            size={240}
+          >
+            <Box padding={3}>
+              <Flex alignItems="center" direction="column" gap={4}>
+                <Text color="white" align="center">
+                  New look! Click Created to see Pins you've published. Click Saved to see your saved Pins and boards.
+                </Text>
+                <Flex alignItems="stretch" direction="column" gap={2}>
+                  <Button
+                    color="white"
+                    size="lg"
+                    text="Got it"
+                    fullWidth
+                  />
+                  <Button
+                    color="semiTransparentWhite"
+                    size="lg"
+                    text="Learn more"
+                    fullWidth
+                  />
+                </Flex>
+              </Flex>
+            </Box>
+          </Popover>
+        </Layer>}
+    </ScrollBoundaryContainer>
+  )
+}
 `}
-  />,
-);
-
-card(
-  <Example
-    name="Selected state"
-    id="selected"
-    defaultCode={`
-function Example() {
-  const [selected, setSelected] = React.useState(false);
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Icons"
+      description={`
+1. Icon end
+Adds an icon after the Button text. Icons should only be used to reinforce a specific function or interaction of the Button. Menus and external links are a common use case. Use \`arrow-up-right\` when linking to an external URL or \`arrow-down\` when displaying a Popover on click. Note that iconEnd on Button is not accessible to screen readers.
```

Comment:
to reinforce

to _visually?_ reinforce

Reply:
`Note that iconEnd on Button is not accessible to screen readers. 
`
Not sure if this is needed.

Reply:
Done! If I recall, Ayesha suggested to add the bit about iconEnd not being accessible to screen readers

Reply:
I think it's probably worth leaving for now, until we limit iconEnd to only a few options.

Reply:
👍